regression/polus-basic-flatfield-correction-plugin/src/basic.py

```diff
@@ -41,10 +41,8 @@ def _get_resized_image_stack(flist):
     
     if len(flist) > OPTIONS['n_sample']:
         N = OPTIONS['n_sample']
-        samples = np.random.randint(len(flist),
-                                    size=(N,),
-                                    replace=False).tolist()
-        flist = [flist[s] for s in samples]
+        samples = np.random.permutation(len(flist)).tolist()
```

Comment:
I don't see any documentation in the function docstring or otherwise as to the purpose of subsampling `flist` based on `OPTIONS['n_sample']`. Can you please explain why this is done?

Reply:
Unfortunately, I also didn't find the long description of this function helpful. I can't leave a direct comment on it since it was not modified. Can you try rewording?